lib/cli/src/extract.ts

```diff
@@ -17,28 +17,7 @@ const read = async (url: string) => {
   const data = JSON.parse(
     await page.evaluate(async () => {
       // eslint-disable-next-line no-undef
-      const d = window.__STORYBOOK_STORY_STORE__.extract();
-
-      const result = Object.entries(d).reduce(
-        (acc, [k, v]: [string, any]) => ({
-          ...acc,
-          [k]: {
-            ...v,
-            parameters: {
-              globals: v.parameters.globals,
-              globalTypes: v.parameters.globalTypes,
-              options: v.parameters.options,
-              args: v.parameters.args,
-              argTypes: v.parameters.argTypes,
-              framework: v.parameters.framework,
-              fileName: v.parameters.fileName,
-              docsOnly: v.parameters.docsOnly,
-            },
-          },
-        }),
-        {}
-      );
-      return JSON.stringify(result, null, 2);
+      return JSON.stringify(window.__STORYBOOK_STORY_STORE__.getDataForManager(), null, 2);
```

Comment:
I thought we were (previously) being careful not to pull out *all* the parameters?

Reply:
Yes... Ow right, parameters like `storysource` are going to make it in now..